src/services/DelegationsService.ts

```diff
@@ -172,6 +175,42 @@ class DelegationsService {
 
     return delegation.store(selectedIdentity)
   }
+
+  public static async checkTraversalStepsToParent(
+    attester: Identity,
+    attestation: Attestation
```

Comment:
I'm surprised we don't have that in the SDK

Reply:
We do now!